docs/src/pages/components/data-grid/localization/localization.md

```diff
@@ -60,6 +60,7 @@ The same import works with `XGrid` as it's an extension of `DataGrid`.
 | Portuguese (Brazil)     | pt-BR               | `ptBR`      |
 | Spanish (Spain)         | es-ES               | `esES`      |
 | Ukraine                 | uk-UA               | `ukUA`      |
+| Italian                 | it-IT               | `itIT`      |
```

Comment:
Please keep the table sorted ASC, same for the imports

Reply:
ok, moving Dutch line too (already not sorted)